kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/Configuration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Kin Ecosystem
+ * Apis for client to server interaction
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+
+package com.kin.ecosystem;
+
+import com.kin.ecosystem.network.ApiClient;
+
+public class Configuration {
+    private static KinEnvironment environment = Environment.getPlayground();
+
+    private static final Object apiClientLock = new Object();
+	private static ApiClient defaultApiClient;
+
+    /**
+     * Get the default API client, which would be used when creating API
+     * instances without providing an API client.
+     *
+     * @return Default API client
+     */
+    public static ApiClient getDefaultApiClient() {
+    	if(defaultApiClient == null) {
+    		synchronized (apiClientLock) {
+    			defaultApiClient = new ApiClient();
+			}
+		}
+        return defaultApiClient;
+    }
+
+	public static KinEnvironment getEnvironment() {
+		return environment;
+	}
+
+	public static void setEnvironment(KinEnvironment environment) {
+		Configuration.environment = environment;
```

Comment:
what will happen if I started with one environment and switch to another, do we go through the registration flow again against the new ecosystem service and blockchain?